resources/test/rpc_schema_hashing_V2.json

```diff
@@ -41,7 +41,7 @@
             "type": "object"
           },
           "AccountHash": {
-            "description": "Hex-encoded account hash.",
+            "description": "Checksummed hex-encoded account hash.",
```

Comment:
Seems like many of the changes in this file are invalid.  However, turns out we're not running the test using this input file in CI unfortunately.

Please fix up this file by running
```
cargo t -p casper-node --lib components::rpc_server::tests::schema --no-default-features
```

Reply:
Fixed file as requested